test/test_rt2.jl

```diff
@@ -0,0 +1,23 @@
+using CompScienceMeshes
+using BEAST
+
+using Test
+
+"""
```

Comment:
This should be a comment, not a docstring. It is currently causing the test suite to fail.